editor/src/components/canvas/controls/property-target-selector.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react'
+import {
+  createLayoutPropertyPath,
+  LayoutTargetableProp,
+} from '../../../core/layout/layout-helpers-new'
+import { getSimpleAttributeAtPath } from '../../../core/model/element-metadata-utils'
+import { bimapEither, eitherToMaybe, left, right } from '../../../core/shared/either'
+import { ElementInstanceMetadata } from '../../../core/shared/element-template'
+import { useColorTheme } from '../../../uuiui'
+
+interface PropertyTargetSelectorProps {
+  targetComponentMetadata: ElementInstanceMetadata | null
+  top: number
+  left: number
+  options: LayoutTargetableProp[]
+  selected: number
+  setOptionsCallback: (options: Array<LayoutTargetableProp>) => void
+}
+
+export const PropertyTargetSelector = (props: PropertyTargetSelectorProps): JSX.Element => {
```

Comment:
`betterReactMemo` please

Reply:
Fixed in 5b68dd64a689f218fbe4e203d4db7b5a9433a636.